com.unity.render-pipelines.high-definition/Runtime/Lighting/AtmosphericScattering/Fog.cs

```diff
@@ -200,4 +218,71 @@ public sealed class FogColorParameter : VolumeParameter<FogColorMode>
         public FogColorParameter(FogColorMode value, bool overrideState = false)
             : base(value, overrideState) { }
     }
+
+    /// <summary>
+    /// This enum defines the two modes for controlling the quality and cost of the volumetric fog.
+    /// </summary>
+    public enum FogControl
+    {
+        /// <summary>
+        /// In this mode, the user changes the parameters on higher abstraction level centered around performance.
+        /// </summary>
+        Balance,
+
+        /// <summary>
+        /// In this mode, the user has a direct access to the internal parameters for the volumetric fog.
+        /// </summary>
+        Manual
+    }
+
+    /// <summary>
+    /// A <see cref="VolumeParameter"/> that holds a <see cref="ExposureMode"/> value.
+    /// </summary>
+    [Serializable]
+    public sealed class FogControlParameter : VolumeParameter<FogControl>
+    {
+        /// <summary>
+        /// Creates a new <see cref="FogControlParameter"/> instance.
+        /// </summary>
+        /// <param name="value">The initial value to store in the parameter.</param>
+        /// <param name="overrideState">The initial override state for the parameter.</param>
+        public FogControlParameter(FogControl value, bool overrideState = false) : base(value, overrideState) { }
+    }
+
+    /// <summary>
+    /// This enum defines which denoising algorithms should be used on the volumetric fog signal.
+    /// </summary>
+    public enum FogDenoisingMode
+    {
+        /// <summary>
+        /// In this mode, the volumetric fog is not filtered.
+        /// </summary>
+        None = 0,
+        /// <summary>
+        /// This technique re-projects data from previous frames to denoise the signal. While being very effective, it can lead to severe ghosting for highly dynamic lighting.
```

Comment:
Use this mode to reproject data from previous frames to denoise the signal. This is effective for static lighting, but it can lead to severe ghosting artifacts for highly dynamic lighting.